spec/unit/attribute_spec.rb

```diff
@@ -94,6 +94,13 @@
     end
   end
 
+  describe '#aliased' do
+    it 'can alias an previously aliased attribute' do
+      expect(users[:id].as(:uid).as(:uuid).sql_literal(ds)).
+        to eql(%("users"."id" AS "uuid"))
+    end
+  end
```

Comment:
It would be good to extract this into a separate PR. I think it's a good fix that can be shipped separately.